sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeLinkedEntityAsyncClient.java

```diff
@@ -49,13 +52,25 @@
         this.service = service;
     }
 
-    Mono<Response<RecognizeLinkedEntitiesResult>> recognizeLinkedEntitiesWithResponse(String text, String language,
-        Context context) {
+    Mono<PagedResponse<com.azure.ai.textanalytics.models.LinkedEntity>> recognizeLinkedEntitiesWithResponse(
```

Comment:
just curious, is there a reason why it is not using the import?

Reply:
same class name is been taken from autogenerated class:
`import com.azure.ai.textanalytics.**implementation**.models.LinkedEntity;`

So can't use import. We transform explored class to autogenerated class, vise versa.